magento/config.php

```diff
@@ -43,6 +43,90 @@ class dahl_dev_config
     protected $_staticUrls = array();
 
     /**
+     * Registered external modules to be loaded
+     * 
+     * @var array
+     * @access protected
+     */
+    protected $_registeredModules = array();
+
+    /**
+     * Register external module to be loaded
+     * 
+     * @param string $path  The path to the root where files are located.
+     * @param stirng $url   An url from where static files can be loaded.
+     * @access public
+     * @return void
+     */
+    public function enableExternal($path, $skinUrl = null)
+    {
+        if (is_array($path)) {
+            foreach ($path as $p) {
+                return $this->enableExternal($p, $skinUrl);
```

Comment:
If you want to enable / disable multiple modules by passing an array you can't return here. Only the first element of the array will be processed.